### PR TITLE
WEBDEV-5417 Search results quantity label doesn't pluralize correctly

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5417

### CODE REVIEW

File: test/collection-browser.test.ts [hunk 1]
```diff
@@ -5,10 +5,28 @@ import sinon from 'sinon';
...
+    await el.updateComplete;
+    el.clearFilters();
+
+    expect(el.selectedFacets).to.equal(defaultSelectedFacets);
+    expect(el.selectedSort).to.equal('relevance');
+    expect(el.sortDirection).to.null;
+    expect(el.sortParam).to.null;
+    expect(el.selectedCreatorFilter).to.null;
+    expect(el.selectedTitleFilter).to.null;
+  });
```
Comment (nsharma123) [inline test/collection-browser.test.ts:28]: Trying to improve code coverage.

File: test/collection-browser.test.ts [hunk 2]
```diff
@@ -5,10 +5,28 @@ import sinon from 'sinon';
 import type { InfiniteScroller } from '@internetarchive/infinite-scroller';
 import type { CollectionBrowser } from '../src/collection-browser';
 import '../src/collection-browser';
+import { defaultSelectedFacets, SortField } from '../src/models';
 import { MockSearchService } from './mocks/mock-search-service';
 import { MockCollectionNameCache } from './mocks/mock-collection-name-cache';
 
 describe('Collection Browser', () => {
+  it('clear existing filter for facets & sort-bar', async () => {
```
Comment (mmcnl) [inline test/collection-browser.test.ts:13]: I'm not clear how this change relates to the plurization ticket?

File: test/collection-browser.test.ts [hunk 3]
```diff
@@ -38,6 +56,9 @@ describe('Collection Browser', () => {
     await el.updateComplete;
 
     expect(searchService.searchParams?.query).to.equal('collection:foo');
+    expect(
+      el.shadowRoot?.querySelector('#big-results-label')?.textContent
+    ).to.contains('Results');
```
Comment (mmcnl) [inline test/collection-browser.test.ts:61]: Is there a corresponding test for the single 'Result' case?

Comment (bfalling) [reply test/collection-browser.test.ts:61]: +1

**Review by bfalling — COMMENTED** [review]
Code looks fine. As @mmcnl implies, I'd add a test for the single and multiple result cases, since they were the main being addressed by the ticket.

File: test/collection-browser.test.ts (hunk 2 above)
Comment (nsharma123) [reply test/collection-browser.test.ts:13]: Trying to imporve code coverage for other parts

**Review by dualcnhq — APPROVED** [review]
lgtm

**Pull request merged**